### ISSUE TITLE
Windows 7 non reporting F keys

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Valyrox/lanterna#96]
```
Under Windows 7 pressing any of the F keys doesn't create a Key instance 
reported by screen.readInput
```

Original issue reported on code.google.com by `rlogia...@gmail.com` on 9 Dec 2013 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
Are you using SwingTerminal?
```

Original comment by `mab...@gmail.com` on 16 Feb 2014 at 7:07
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Not explicitly, here is what I'm using to start it:

                final Screen screen = TerminalFacade.createScreen();
        screen.startScreen();

```

Original comment by `rlogia...@gmail.com` on 16 Feb 2014 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
Okay, I think this is the same issue as was reported by another user a little 
while ago. The F1-F12 keys were not properly detected in SwingTerminal. There 
is a fix for this on the 2.1.x-branch, but no release yet. If you can try the 
2.1.8-SNAPSHOT build, please do, otherwise be a bit patient and I'll have 
another release done in a few weeks or so.
```

Original comment by `mab...@gmail.com` on 17 Feb 2014 at 12:10

**Comment by GoogleCodeExporter** [comment]
```
This issue has been closed because Google Code is shutting down shortly and all 
issues have been migrated, along with the rest of the project, to GitHub. 
However, notifications will probably not work when updates are made to this 
issue on GitHub so for future updates, please visit 
https://github.com/mabe02/lanterna/issues and subscribe to further updates 
there.

Thank you,
Martin
```

Original comment by `mab...@gmail.com` on 12 Jul 2015 at 7:39
- Changed state: **Duplicate**
